data/SandipPalit/events/2023-02-11-TFUG-Kolkata-Reboot.json

```
@@ -0,0 +1,12 @@
+{
+  "isVirtual": false,
+  "isInPerson": true,
+  "color": "orange",
+  "name": "TFUG Kolkata Reboot",
+  "description": "The purpose of this event is to bring together TensorFlow enthusiasts, professionals, and experts to share their knowledge, experiences, and insights about the latest developments in TensorFlow.",
+  "date": {
+    "start": "2023-02-11T00:00:00.000+00:00",
+    "end": "2023-02-12T00:00:00.000+00:00"
+  },
+  "url": "https://twitter.com/TFUGKol/status/1623582121653837825?s=20&t=-LMGv2gn1TYPXHKtt4mc6g"
```

You might want to remove the unneeded part of the link.
```suggestion
  "url": "https://twitter.com/TFUGKol/status/1623582121653837825"
```